backend/overstroomik_service/main.py

```diff
@@ -34,12 +35,19 @@ async def read_root():
 @app.get('/by_rd', response_model=FloodInfo)
 async def by_rd(x: Optional[float] = None, y: Optional[float] = None) -> FloodInfo:
 
-    # Get location information from the geoserver
-    data = Data()
+    # determine which pdok function by input
+    if x and y:   
+        # Get location information from the geoserver
+        geoserver = Geoserver()
```

Comment:
You never really use the class itself. All the functions could be `@staticmethods` and then just call `Geoserver.get_data(x, y)`?